chitchat/src/delta.rs

```diff
@@ -16,6 +17,10 @@ impl Serializable for Delta {
             node_id.serialize(buf);
             node_delta.serialize(buf);
         }
+        (self.nodes_to_reset.len() as u16).serialize(buf);
```

Comment:
```suggestion
       u16::try_from(self.nodes_to_reset.len()).unwrap().serialize(buf);
```

would be a little bit more defensive.